src/routes/our-music/index.svelte

```diff
@@ -1,3 +1,130 @@
+<script>
+    import { audioData } from './audioData.js';
+
+    import TrackHeading from './TrackHeading.svelte';
+    import TrackHeader from './TrackHeader.svelte';
+    import ProgressBarTime from './ProgressBarTime.svelte';
+    import Controls from './Controls.svelte';
+    import VolumeSlider from './VolumeSlider.svelte';
+    import PlayList from './PlayList.svelte';
```

Comment:
Good effort in separating them into smaller components 👍 

Nonetheless, our-music page is still getting super complicated. There are a lot of logic handled inside and it results in this page having 400+ lines of code :').

To make it easier to maintain, can we try separating the whole playlist logic into a separate component instead? For me, it would be better to merge all these smaller components into another component before finally putting the big component inside of our-music page. Just let me know if you have a different suggestion.

Reply:
For now, we keep it this way because some functions are needed in different files, hence putting in index.svelte is the best currently. We'll see as the next development will be much much later.